sequential.py

```diff
@@ -0,0 +1,32 @@
+import re
+import os
```

Comment:
This is a plain python script.
Please use all the method demoed in our session to make it look more like a python project